contrib/tcpdump/netdissect.h

```diff
@@ -143,7 +143,7 @@ typedef struct netdissect_options netdissect_options;
 
 typedef u_int (*if_printer) IF_PRINTER_ARGS;
 
-#if __has_feature(capabilities) && defined(__mips__)
+#if __has_feature(capabilities) && defined(__mips__) && defined(CHERI_TCPDUMP)
```

Comment:
```suggestion
#if __has_feature(capabilities) && defined(CHERI_TCPDUMP)
```

Reply:
This is not MIPS-specific right? We just want to disable it since it's broken?

Reply:
Removed the mips bit